// hero_data/lib/hero_data.js
exports.heroData = [{
  name : "Tidehunter",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/tidehunter_full.png",
  abilities : [{
    name : "Gush",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/tidehunter_gush_hp1.png",
    cooldowns : 12,
  }, {
    name : "Kraken Shell",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/tidehunter_kraken_shell_hp1.png",
    isPassive : true,
  }, {
    name : "Anchor Smash",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/tidehunter_anchor_smash_hp1.png",
    cooldowns : [7, 6, 5, 4],
  }, {
    name : "Ravage",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/tidehunter_ravage_hp1.png",
    cooldowns : 150,
    isUlti : true,
  }],
}, {
  name : "Bane",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/bane_full.png",
  abilities : [{
    name : "Enfeeble",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/bane_enfeeble_hp1.png",
    cooldowns : 10,
  }, {
    name : "Brain Sap",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/bane_brain_sap_hp1.png",
    cooldowns : 14,
  }, {
    name : "Nightmare",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/bane_nightmare_hp1.png",
    cooldowns : 15,
  }, {
    name : "Fiend's Grip",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/bane_fiends_grip_hp1.png",
    cooldowns : 100,
    isUlti : true,
  }],
}, {
  name : "Sand King",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/sand_king_full.png",
  abilities : [{
    name : "Burrowstrike",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/sandking_burrowstrike_hp1.png",
    cooldowns : 11,
  }, {
    name : "Sand Storm",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/sandking_sand_storm_hp1.png",
    cooldowns : [40, 30, 20, 10],
    isPassive : true,
  }, {
    name : "Caustic Finale",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/sandking_caustic_finale_hp1.png",
    isPassive : true,
  }, {
    name : "Epicenter",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/sandking_epicenter_hp1.png",
    cooldowns : [140, 120, 100],
  }],
}, {
  name : "Enigma",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/enigma_full.png",
  abilities : [{
    name : "Malefice",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/enigma_malefice_hp1.png",
    cooldowns : 15,
  }, {
    name : "Demonic Conversion",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/enigma_demonic_conversion_hp1.png",
    cooldowns : 35,
  }, {
    name : "Midnight Pulse",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/enigma_midnight_pulse_hp1.png",
    cooldowns : 25,
  }, {
    name : "Black Hole",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/enigma_black_hole_hp1.png",
    cooldowns : [200, 190, 180],
  }],
}, {
  name : "Doom",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/doom_bringer_full.png",
  abilities : [{
    name : "Devour",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/doom_bringer_devour_hp1.png",
    cooldowns : [70, 60, 50, 40],
  }, {
    name : "Scorched Earth",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/doom_bringer_scorched_earth_hp1.png",
    cooldowns : [60, 55, 50, 45],
  }, {
    name : "LVL? Death",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/doom_bringer_lvl_death_hp1.png",
    cooldowns : 8,
  }, {
    name : "Doom",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/doom_bringer_doom_hp1.png",
    cooldowns : 100,
    isUlti : true,
  }],
}, {
  name : "Earthshaker",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/earthshaker_full.png",
  abilities : [{
    name : "Fissure",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/earthshaker_fissure_hp1.png",
    cooldowns : 15,
  }, {
    name : "Enchant Totem",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/earthshaker_enchant_totem_hp1.png",
    cooldowns : 5,
  }, {
    name : "Aftershock",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/earthshaker_aftershock_hp1.png",
    isPassive : true,
  }, {
    name : "Echo Slam",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/earthshaker_echo_slam_hp1.png",
    cooldowns : [150, 130, 110],
  }],
}, {
  name : "Magnus",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/magnataur_full.png",
  abilities : [{
    name : "Shockwave",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/magnataur_shockwave_hp1.png",
    cooldowns : [10, 9, 8, 7],
  }, {
    name : "Empower",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/magnataur_empower_hp1.png",
    cooldowns : 12,
  }, {
    name : "Skewer",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/magnataur_skewer_hp1.png",
    cooldowns : 30,
  }, {
    name : "Reverse Polarity",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/magnataur_reverse_polarity_hp1.png",
    cooldowns : [120, 110, 100],
  }],
}, {
  name : "Silencer",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/silencer_full.png",
  abilities : [{
    name : "Curse Of Silent",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/silencer_curse_of_the_silent_hp1.png",
    cooldowns : [20, 16, 12, 8],
  }, {
    name : "Glaives Of Wisdom",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/silencer_glaives_of_wisdom_hp1.png",
    cooldowns : 1,
  }, {
    name : "Last Word",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/silencer_last_word_hp1.png",
    cooldowns : [36, 28, 20, 12],
  }, {
    name : "Global Silence",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/silencer_global_silence_hp1.png",
    cooldowns : 130,
    isUlti : true,
  }],
}, {
  name : "Outworld Devourer",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/obsidian_destroyer_full.png",
  abilities : [{
    name : "Arcane Orb",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/obsidian_destroyer_arcane_orb_hp1.png",
    cooldowns : 1,
  }, {
    name : "Astral",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/obsidian_destroyer_astral_imprisonment_hp1.png",
    cooldowns : [18, 16, 14, 12],
  }, {
    name : "Essense Aura",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/obsidian_destroyer_essence_aura_hp1.png",
    isPassive : true,
  }, {
    name : "Sanity Eclipse",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/obsidian_destroyer_sanity_eclipse_hp1.png",
    cooldowns : 160,
    isUlti : true,
  }],
}, {
  name : "Wraith King",
  img : "http://cdn.dota2.com/apps/dota2/images/heroes/skeleton_king_full.png",
  abilities : [{
    name : "Wraithfire Blast",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/skeleton_king_hellfire_blast_hp1.png",
    cooldowns : 8,
  }, {
    name : "Vampiric Aura",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/skeleton_king_vampiric_aura_hp1.png",
    isPassive : true,
  }, {
    name : "Mortal Strike",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/skeleton_king_mortal_strike_hp1.png",
    isPassive : true,
  }, {
    name : "Reincarnation",
    img : "http://cdn.dota2.com/apps/dota2/images/abilities/skeleton_king_reincarnation_hp1.png",
    cooldowns : [260, 160, 60],
    isPassive : true,
  }],
}];
